blazor/scheduler/timescale.md

```diff
@@ -52,6 +52,47 @@ The `Interval` and `SlotCount` properties can be used together on the Scheduler
 
 ![Time Slot Duration in Blazor Scheduler](images/blazor-scheduler-timeslot.png)
 
+## Setting different time slot duration for each view
+
+The `Interval` and `SlotCount` properties within the `ScheduleView` tag as child content of the `ScheduleViewTimeScale` to establish distinct time slot durations for various views, as demonstrated in the provided code snippet. In this illustration, the `Day` view encompasses six time slots per hour, while the `Week` and `WorkWeek` views comprise two time slots per hour.
```

Comment:
By using the `Interval` and `SlotCount` properties, within the `ScheduleViewTimescale` tag as a child element of `ScheduleView` you can customize the duration of time slots for views. Take a look at the code snippet to see how it's done. In this example, the `Day` view has six-time slots per hour while the `Week` and `WorkWeek` views have two-time slots, per hour.

Reply:
Content included